src/cmd/server.ts

```diff
@@ -64,6 +58,15 @@ async function main(config: Config) {
   }
 
   client.once('ready', () => {
+    client.guilds.valueOf().forEach( guild => {
+      try {
+        deleteCommands(config, guild.id);
+        createCommands(config, guild.id);
+      } catch (error) {
+        console.error(`Could not deploy commands to ${guild.id}! -- ${guild.name}`);
+      }
+    });
+
```

Comment:
Is the intent to delete/create slash commands for each guild sequentially, or do we want to delete/create slash commands concurrently? If the goal is the latter, then we want something like this instead:

```suggestion
  client.once('ready', async () => {
    await Promise.all(client.guilds.valueOf().map( async guild => {
      try {
        await deleteCommands(config, guild.id);
        await createCommands(config, guild.id);
      } catch (error) {
        console.error(`Could not deploy commands to ${guild.id}! -- ${guild.name}`);
      }
    }));
```